spec/schema.json

```diff
@@ -249,11 +249,14 @@
         "law_governed": {
           "type": "string"
         },
-        "public_register_name": {
+        "register_name": {
```

Comment:
Changing this is fine, but it will have a knock on effect in 'chips-filing-consumer' which has already mapped this data and is expecting a field named 'public_register_name'. I can't remember, but it may even fall over if the field name changes.

Reply:
Also, the Web entity model has been built using the public_register_name so that would also need changing, probably best to leave as public_register_name

Reply:
Corporate BOs aren't implemented yet in `chips-filing-consumer` so changes there are still ok. If web, api and spec are all in sync then this should be fine.